src/terminal/parser/OutputStateMachineEngine.hpp

```diff
@@ -230,6 +231,10 @@ namespace Microsoft::Console::VirtualTerminal
         bool _GetRepeatCount(const std::basic_string_view<size_t> parameters,
                              size_t& repeatCount) const noexcept;
 
+        bool OutputStateMachineEngine::_GetOscSetClipboard(const std::wstring_view string,

```

Comment:
Is there a reason why the `OutputStateMachineEngine::` class prefix is included here in the header?

Reply:
Oh, my silly mistake. I copied that directly from the implementation. 😂  Already fixed.